src/test/java/guitests/HideCommandTest.java

```diff
@@ -87,29 +87,29 @@ public void hide_type_completedEvents() {
     }
     
     @Test
-    public void hide_date_on1Jan2018() {
-        assertListResult("hide on 1 jan 2018", td.submitPrototype, 
+    public void hide_date_on1Jan2022() {
+        assertListResult("hide on 1 jan 2022", td.submitPrototype, 
                 td.submitProgressReport, td.developerMeeting,
                 td.learnVim, td.buyBirthdayGift, td.signUpForYoga);
     }
     
     @Test
-    public void hide_deadline_byYear2016() {
-        assertListResult("hide by 12 dec 2016", td.submitPrototype, 
+    public void hide_deadline_byYear2020() {
+        assertListResult("hide by 12 dec 2020", td.submitPrototype, 
```

Comment:
@initialshl the test is going to fail again in 10 days, need a new PR plz :sob:

<sub>lolol</sub>